bin/cocos_project.py

```diff
@@ -281,8 +281,11 @@ def _gen_available_platforms(self, proj_info):
             else:
                 cfg_obj = cfg_class(root_path, self._project._is_script_project())
 
-            if cfg_obj._is_available():
-                self._available_platforms[p] = cfg_obj
+            # if cfg_obj._is_available():
+            #     self._available_platforms[p] = cfg_obj
+
+            # do not verify available
```

Comment:
So, you should implement: when `--proj-dir` is used, it should be check the specified project dir, but not the default one.